dataactvalidator/filestreaming/csv_selection.py

```diff
@@ -66,36 +67,40 @@ def write_query_to_file(local_filename, upload_name, header, file_type, is_local
     if is_local:
         local_filename = upload_name
 
-    # create file locally
+    # get the raw SQL equivalent
+    raw_query = generate_raw_quoted_query(query_func(query_utils))
+    # save psql command with query to a temp file
+    temp_sql_file, temp_sql_file_path = generate_temp_query_file(raw_query)
+
+    log_data = {
```

Comment:
If you aren't doing anything else with this variable it should just be inside `logger.debug` rather than being a variable